lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -336,6 +341,7 @@ def main():
         override_display=dict(default=None),
         kernel_params=dict(default=None, type='list'),
         hosted_engine=dict(default=None, choices=['deploy', 'undeploy']),
+        iscsi=dict(default=None, type='dict'),
```

Comment:
Can you rename `iscsi` to `iscsi_address`, I think it's not needed to have it a dict, no? String should be fine.

Also please add

```python
required_if=[
  ['state', 'iscsidiscover', ['iscsi_address']]
]
```

And also please document this parameter.